cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/default-service/AwsDefaults.service.tsx

```diff
@@ -0,0 +1,15 @@
+// React imports
+import React from 'react';
+import { AwsService, AWSVirtualMachine } from '../../../../../swagger-api';
+
+export class AwsDefaults extends React.Component {
+    // Find what the default osImage is
+    static selectDefalutOsImage = (osImages: AWSVirtualMachine[]) => {
+        return osImages[0];
+    };
+
+    // Retrieve all osimages
```

Comment:
not needed in this service, this should be done in the AwsOrchestrator around line 16

Reply:
Got it